src/material/core/option/option.html

```diff
@@ -1,5 +1,6 @@
-<mat-pseudo-checkbox *ngIf="multiple" class="mat-mdc-option-pseudo-checkbox"
```

Comment:
I suspect that it defaults to `accent`, because `mat-checkbox` does too and it was written to match `mat-checkbox` as closely as possible. We could leave on the default and have the consuming component override the color. 

That being said, I found the current color a little weird since your select screenshot looks right, but the autocomplete does not.

Reply:
I changed this to use a mathematic formula to place the checkmark. Hopefully this fixes the centering. Does it look centered to you? I can barely tell the difference between before and after.

Reply:
For 2, I sent out #25983 to fix autocomplete not applying the right theme to the panel.

Reply:
does this PR include the changes from the PR you linked? I still see the autocomplete check showing up as the wrong color

Reply:
#25983 landed, and I believe the appearance for the autocomplete has been corrected. Could someone please confirm that this is the correct appearance? Is the font color supposed to change when there's a checkmark? See attached screenshots of both with and without selection indicators.

<img width="451" alt="image" src="https://user-images.githubusercontent.com/7720245/210837184-4bc94745-c17a-4b36-ba47-98b06c3ee0e4.png">
<img width="456" alt="image" src="https://user-images.githubusercontent.com/7720245/210837219-068d4d0b-76ae-49dc-a070-09748b71b9fe.png">